tensorflow/training/docker/2.6/py3/cu112/Dockerfile.gpu

```
@@ -0,0 +1,330 @@
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+ENV RDMAV_FORK_SAFE=1
+
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.8.2
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+ARG EFA_VERSION=1.12.1
+ARG OMPI_VERSION=4.1.1
+ARG BRANCH_OFI=1.1.3-aws
+
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.6_aws/20210816_235124/gpu/cu112/py38/tensorflow_gpu-2.6.0-cp38-cp38-manylinux2010_x86_64.whl
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.12
+
+ARG SMDATAPARALLEL_BINARY=https://smdataparallel.s3.amazonaws.com/binary/tensorflow/2.6.0/cu112/2021-08-26/smdistributed_dataparallel-1.2.1-cp38-cp38-linux_x86_64.whl
+
+ARG SMMODELPARALLEL_BINARY=https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/tensorflow-2.6.0/build-artifacts/2021-08-24-22-08/smdistributed_modelparallel-1.4.0-cp38-cp38-linux_x86_64.whl
+
+ARG CUDA=11.2
+ARG CUDA_DASH=11-2
+ARG CUDNN=8.1.0.77-1
+ARG NCCL_VERSION=2.8.4
+
+RUN apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated --allow-downgrades  --allow-change-held-packages \
+   ca-certificates \
+   cuda-command-line-tools-${CUDA_DASH} \
+   cuda-cudart-dev-${CUDA_DASH} \
+   libcufft-dev-${CUDA_DASH} \
+   libcurand-dev-${CUDA_DASH} \
+   libcusolver-dev-${CUDA_DASH} \
+   libcusparse-dev-${CUDA_DASH} \
+   curl \
+   emacs \
+   libcudnn8=${CUDNN}+cuda${CUDA} \
+   # TensorFlow doesn't require libnccl anymore but Open MPI still depends on it
+   libgomp1 \
+   libfreetype6-dev \
+   libhdf5-serial-dev \
+   liblzma-dev \
+   libpng-dev \
+   libtemplate-perl \
+   libzmq3-dev \
+   hwloc \
+   git \
+   unzip \
+   wget \
+   libtool \
+   vim \
+   libssl1.1 \
+   openssl \
+   build-essential \
+   openssh-client \
+   openssh-server \
+   zlib1g-dev \
+   # Install dependent library for OpenCV
+   libgtk2.0-dev \
+   jq \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends --allow-unauthenticated --allow-change-held-packages \
+   libcublas-dev-${CUDA_DASH} \
+   libcublas-${CUDA_DASH} \
+   # The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0
+   # adds a new list which contains libnvinfer library, so it needs another
+   # 'apt-get update' to retrieve that list before it can actually install the
+   # library.
+   # We don't install libnvinfer-dev since we don't need to build against TensorRT,
+   # and libnvinfer4 doesn't contain libnvinfer.a static library.
+   # nvinfer-runtime-trt-repo doesn't have a 1804-cuda10.1 version yet. see:
+   # https://developer.download.nvidia.cn/compute/machine-learning/repos/ubuntu1804/x86_64/
+ && rm -rf /var/lib/apt/lists/* \
+ && mkdir -p /var/run/sshd
```

Please add `apt-get clean` here.
```suggestion
 && mkdir -p /var/run/sshd \
 && apt-get clean
```

---

done